src/pages/ItemList.js

```diff
@@ -124,6 +124,17 @@ function ItemList({ token }) {
     setSearchInput(e.target.value);
   };
 
+  const handleDelete = async (id, item_name) => {
+    const confirmation = window.confirm(
+      `Are you sure you want to delete ${item_name}?`,
```

Comment:
I of course know about alert() but wasn't familiar with confirm() and how you can use it as a condition! This is v cool

Reply:
Agree that this is super cool!